core/renderers/zelos/info.js

```diff
@@ -82,6 +83,13 @@ Blockly.zelos.RenderInfo = function(renderer, block) {
    * @override
    */
   this.isInline = true;
+
+  /**
+   * An object with rendering information about the right connection shape.
+   * @type {!Blockly.zelos.RightConnectionShape}
```

Comment:
Fix type: this is very definitely nullable.

Reply:
Yep, just fixed travis.